db/migrate/20170421215941_create_submission_packets.rb

```diff
@@ -0,0 +1,9 @@
+class CreateSubmissionPackets < ActiveRecord::Migration[5.0]
+  def change
+    create_table :submission_packets do |t|
+      t.references :poem, foreign_key: true, index: true, null: false
+      t.references :submission, foreign_key: true, index: true, null: false
+      t.string :status
```

Comment:
That's an interesting idea, but I think if anything can be calculated. I don't think I can calculate the `SubmissionPacket#status` but I might be able to nix `Poem#status` and calculate that instead. 


I'll add a card to figure that out and table that for another PR. For now, I'll just update these to default to "Submitted".

Thanks to you and @gylaz  for the push here. It's helping me clarify the recs.